src/components/Article/Image.tsx

```diff
@@ -3,21 +3,26 @@ type ImageProps = {
 };
 
 export const Image = ({ article }: ImageProps) => {
+  console.log(article.cover_image)
   return (
-    <div className="h-full w-full absolute pointer-events-none">
-      <div
-        className="h-full w-full absolute blur-md brightness-50 -z-20"
-        style={{
-          backgroundImage: `url(${article.cover_image})`,
-          backgroundSize: "contain",
-          backgroundPosition: "center",
-        }}
-      />
-      <img
-        className="h-full w-full max-w-screen absolute -top-20 object-contain -z-10"
-        src={article.cover_image}
-        alt={article.title}
-      />
-    </div>
+    <>
```

Comment:
i think a ternary that returns null is cleaner than a fragment, it's more explicit, while the && in this context gives an impression that other things might be returned apart from it